src/viur/core/module.py

```diff
@@ -81,6 +82,8 @@ def parse_value_by_annotation(annotation: type, name: str, value: str | list | t
             Tries to parse a value according to a given type.
             May be called recursively to handle unions, lists and tuples as well.
             """
+            # logging.debug(f"{annotation=} | {name=} | {value=}")
+
```

Comment:
```suggestion
```
I think we dont't need this.

Reply:
I don't care, I find it quite practical if you have to debug something and only have to comment it in. But it can also go away. Let @phorward  decide.